ODIN_II/SRC/parse_making_ast.cpp

```diff
@@ -1801,55 +1836,65 @@ void graphVizOutputAst(std::string path, ast_node_t* top) {
 /*---------------------------------------------------------------------------------------------
  * (function: graphVizOutputAst_traverse_node)
  *-------------------------------------------------------------------------------------------*/
-void graphVizOutputAst_traverse_node(FILE* fp, ast_node_t* node, ast_node_t* from, int from_num) {
+void graphVizOutputAst_traverse_node(FILE* fp, ast_node_t* node, ast_node_t* from, int from_num, bool identifier) {
```

Comment:
I think what you have done offers us a really good opportunity to combine identifiers inside a node, hopefully reducing the graph size and improve the readability

I think we should print the identifier as part of the node rather than separately.
To have it pretty print you need to use html tags

so the end-goal should print something along these line

```
[label=<identifier<br/>NODE_TYPE>]
```

and it will render roughly like this

```
,--------------------,
|     identifier     |
|    NODE_TYPE       |
'--------------------'
```

to do so you need to first print `fprintf(fp, "\t%d [label=<", my_label);` first

then 

```
 if (identifier) {
        fprintf(fp, "%s<br/>", node->types.identifier);
 }
```

then do the regular switch

then finish with

```
fprintf(fp, ">];\n"); 
```